services/skipchain/msgs.go

```diff
@@ -33,18 +36,23 @@ func init() {
 // deny a SkipBlock.
 type VerifierID uuid.UUID
 
+// SkipBlockVerifier is function that should return whether this skipblock is
+// accepted or not. This function is used during a BFTCosi round, but wrapped
+// around so it accepts a block.
+type SkipBlockVerifier func(msg []byte, s *SkipBlock) bool
+
+// RegisterVerification stores the verification in a map and will
+// call it whenever a verification needs to be done.
+func RegisterVerification(c *sda.Context, v VerifierID, f SkipBlockVerifier) error {
```

Comment:
I thought keeping it, so that `Identity` or any other skipchain can call `skipchain.RegisterVerification` and give its `sda.Context`, so that it gets stored correctly.